doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -356,7 +356,7 @@ which has the type
                                        OSSL_CMP_MSG **res);
 Returns 1 on success, 0 on error.
 
-Default is NULL, which implies the use of L<OSSL_CMP_MSG_http_perform(3)>.
+Default is NULL, which implies the use of <OSSL_CMP_MSG_http_perform().
```

Comment:
Wait, what???

This doesn't look right...  and looking through the source, I can find no place where the cb is actually used.  But also, isn't there some CMP/CRMF PR hanging around still unmerged?  So I'm thinking that  until that's finally resolved, we must hang back on this page.

Reply:
I disagree.  I think referring to things that don't exist is wrong.  If they ever get written (and I am not very optimistic) then we can backpatch.  There's already an option that separate lists missing pages (the -u/-v flags).

Reply:
The CMP code is still being written and there are a number of PRs in the pipeline that will add further documentation, so I fully expect this to be documented at some point. @DDvO might confirm.

Reply:
Yes, correct! `OSSL_CMP_MSG_http_perform`  will be part of chunk 8.

Reply:
Okay, I restored the link.